File: Atividade004/questao01.py
reprovados = 0
aprovados = 0
menor_media = 10

for i in range(25):
    media = float(input(f"Digite a media do aluno {i+1}: "))

    if media < 0 or media > 10:
        print("Media invalida. Insira um valor entre 0 e 10.")
        continue

    if media < menor_media:
        menor_media = media

    if media >= 6.9:
        aprovados += 1
    else:
        reprovados += 1

print(f"\nQuantidade de alunos aprovados: {aprovados}")
print(f"Quantidade de alunos reprovados: {reprovados}")
print(f"Menor media informada: {menor_media:.2f}")

File: ListaAvaliativa01/questao01.c
#include <stdio.h>

int main() {
    int A, B, C, D, diferenca;

    printf("Digite o valor de A: ");
    scanf("%d", &A);

    printf("Digite o valor de B: ");
    scanf("%d", &B);

    printf("Digite o valor de C: ");
    scanf("%d", &C);

    printf("Digite o valor de D: ");
    scanf("%d", &D);

    diferenca = (A * B - C * D);

    printf("DIFERENCA = (%d * %d - %d * %d)\n", A, B, C, D);
    printf("DIFERENCA = (%d)\n", diferenca);

    return 0;
}